### PR TITLE
Update category page header

### LINKED ISSUE #6741: Update category page header
See: https://mozilla.invisionapp.com/d/main#/console/11679087/256136385/preview

---

We should update the category page header:

- title should be centered
- no description

screen shot 2018-10-24 at 18 20 12

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #6741

~~⚠️ Depends on #6739~~

---

This PR updates the `CategoryHeader` according to the new-ish mock-ups in the issue linked above.

## Screenshots

screen shot 2018-10-25 at 11 19 38
screen shot 2018-10-25 at 11 19 46
screen shot 2018-10-25 at 11 19 55
screen shot 2018-10-25 at 11 19 58
screen shot 2018-10-25 at 11 20 02
screen shot 2018-10-25 at 11 20 08
screen shot 2018-10-25 at 11 21 07
screen shot 2018-10-25 at 11 21 10
screen shot 2018-10-25 at 11 21 15
screen shot 2018-10-25 at 11 21 18
screen shot 2018-10-25 at 11 21 23
screen shot 2018-10-25 at 11 21 29
screen shot 2018-10-25 at 11 21 33

### CODE REVIEW

**Comment by willdurand** [comment]
Marking this PR as WIP again because I think I can convert the file to Flow if https://github.com/mozilla/addons-frontend/pull/6739 gets merged first.

**Comment by willdurand** [comment]
rebased!

File: src/amo/components/CategoryHeader/index.js [hunk 1]
```diff
@@ -1,95 +1,55 @@
...
-          'Browse all extensions in this category.',
-        );
-      }
-
-      if (category.type === ADDON_TYPE_THEME) {
-        category.description = i18n.gettext(
-          'Browse all themes in this category.',
-        );
-      }
+      className = `CategoryHeader--type-${addonType}`;
```
Comment (rebmullin) [inline src/amo/components/CategoryHeader/index.js]: as far as I can tell, there are no styles associated with this classname. Just checking..do we still need this?

Comment (willdurand) [reply src/amo/components/CategoryHeader/index.js]: right let's remove it, thanks!

**Pull request merged**